src/consts/permissions.ts

```diff
@@ -65,51 +64,90 @@ enum PERMISSIONS {
 
   /* -------- EscolaLMS/Courses -------- */
 
-  CourseCreate = 'create course',
-  CourseRead = 'attend course',
-  CourseUpdate = 'update course',
-  CourseDelete = 'delete course',
+  CourseList = 'course_list',
+  CourseCreate = 'course_create',
+  CourseRead = 'course_attend', // <- course_read
+  CourseUpdate = 'course_update',
+  CourseDelete = 'course_delete',
+  CourseUpdateAuthored = 'course_update_authored',
+  CourseDeleteAuthored = 'course_delete_authored',
+  CourseReadAuthored = 'course_attend_authored', // <- course_read_authored
+
+  /* -------- EscolaLMS/H5P -------- */
+
+  H5PList = 'h5p_list',
+  H5PRead = 'h5p_read',
+  H5PCreate = 'h5p_create',
+  H5PDelete = 'h5p_delete',
+  H5PUpdate = 'h5p-update',
+  H5PLibraryList = 'h5p_library_list',
+  H5PLibraryRead = 'h5p_library_read',
+  H5PLibraryCreate = 'h5p_library_create',
+  H5PLibraryDelete = 'h5p_library_delete',
+  H5PLibraryUpdate = 'h5p_library_update',
 
   /* -------- EscolaLMS/payments -------- */
 
-  PaymentRead = 'view payment', // TODO: unused in api
-  PaymentList = 'search all payments', // TODO: unused in api
+  PaymentRead = 'payment_read', // TODO: unused in api
+  PaymentList = 'payment_list', // TODO: unused in api
 
   /* -------- EscolaLMS/Categories -------- */
 
-  CategoryCreate = 'create category',
-  CategoryUpdate = 'update category',
-  CategoryDelete = 'delete category',
+  CategoryList = 'category_list',
+  CategoryRead = 'category_read',
+  CategoryCreate = 'category_create',
+  CategoryUpdate = 'category_update',
+  CategoryDelete = 'category_delete',
 
   /* -------- EscolaLMS/pages -------- */
 
-  PageCreate = 'create pages',
-  PageUpdate = 'update pages',
-  PageDelete = 'delete pages',
+  PageList = 'pages_list',
+  PageRead = 'page_read',
+  PageCreate = 'pages_create',
+  PageUpdate = 'pages_update',
+  PageDelete = 'pages_delete',
 
   /* -------- EscolaLMS/Scorm -------- */
 
-  ScormCreate = 'create Scorm',
-  ScormUpdate = 'update Scorm',
-  ScormDelete = 'delete Scorm',
+  ScormList = 'scorm_list',
+  ScormRead = 'scorm_read',
+  ScormCreate = 'scorm_create',
+  ScormUpdate = 'scorm_update',
+  ScormDelete = 'scorm_delete',
+  ScormGetTrack = 'scorm_get_track',
+  ScormSetTrack = 'scorm_set_track',
```

Comment:
nie wiem czym jest track, jeżeli modelem to `scorm_track_update` jeżeli nie to `scorm_track-update`